2.3 RC1

mambax7 opened the pull request.

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 490.57kb -> 413.60kb (15.69%)

/assets/js/lightbox/images/favicon.gif -- 1.15kb -> 0.13kb (89%)
/assets/images/rssfeed_buttons.png -- 3.00kb -> 0.46kb (84.54%)
/assets/images/plus.png -- 3.14kb -> 0.64kb (79.48%)
/assets/images/admin/add_ads.png -- 3.14kb -> 0.64kb (79.48%)
/assets/images/admin/comment.png -- 3.09kb -> 0.74kb (75.99%)
/assets/images/icons/search_button_16.png -- 3.24kb -> 0.83kb (74.4%)
/assets/images/admin/web_layout.png -- 3.10kb -> 0.79kb (74.36%)
/assets/images/logo_mini.png -- 3.20kb -> 0.87kb (72.74%)
/assets/images/admin/info_button.png -- 3.20kb -> 0.87kb (72.74%)
/assets/images/admin/star_16.png -- 3.29kb -> 0.96kb (70.88%)
/assets/images/admin/email_forward.png -- 3.28kb -> 1.06kb (67.55%)
/assets/js/lightbox/images/bg-checker.png -- 0.93kb -> 0.30kb (67.47%)
/assets/images/admin/comment_32.png -- 3.42kb -> 1.14kb (66.73%)
/assets/images/admin/error_button.png -- 3.53kb -> 1.18kb (66.56%)
/assets/images/admin/error_button_32.png -- 3.53kb -> 1.18kb (66.56%)
/assets/images/admin/comment_user.png -- 3.39kb -> 1.16kb (65.75%)
/assets/images/admin/info_button_32.png -- 3.63kb -> 1.25kb (65.55%)
/assets/images/admin/search_button_green_32.png -- 3.87kb -> 1.47kb (62.05%)
/assets/images/admin/warning_32.png -- 4.13kb -> 1.66kb (59.66%)
/assets/images/admin/page_table_close_32.png -- 4.31kb -> 1.85kb (57.08%)
/assets/images/admin/book_search.png -- 4.34kb -> 1.88kb (56.66%)
/assets/images/icons/discussion.png -- 4.52kb -> 2.05kb (54.76%)
/assets/js/lightbox/images/speech-bubbles.png -- 3.48kb -> 1.79kb (48.72%)
/assets/js/lightbox/images/donate.png -- 5.00kb -> 2.67kb (46.73%)
/maps/french/images/map.png -- 11.20kb -> 6.58kb (41.3%)
/maps/russian/images/map.png -- 13.52kb -> 8.42kb (37.68%)
/assets/images/img_cat/default.png -- 3.91kb -> 2.87kb (26.54%)
/assets/images/menu/edit2.png -- 1.40kb -> 1.09kb (21.88%)
/assets/images/menu/newdoc2.png -- 1.23kb -> 1.00kb (18.95%)
/assets/images/deco/icon.png -- 0.95kb -> 0.79kb (16.43%)
/assets/js/lightbox/images/box.png -- 11.46kb -> 9.93kb (13.3%)
/assets/images/menu/profile.png -- 1.75kb -> 1.54kb (12%)
/assets/images/menu/comment.png -- 1.85kb -> 1.65kb (10.96%)
/assets/images/camera_photo.png -- 1.43kb -> 1.28kb (10.55%)
/assets/images/admin/avatar-iluc.jpg -- 13.18kb -> 12.09kb (8.23%)
/assets/images/img_cat/animals.png -- 16.51kb -> 15.65kb (5.19%)
/assets/images/img_cat/toys.png -- 16.49kb -> 15.65kb (5.12%)
/assets/images/img_cat/furniture.png -- 16.95kb -> 16.11kb (4.98%)
/assets/js/lightbox/images/prev.png -- 1.33kb -> 1.27kb (4.26%)
/assets/images/img_cat/nature.png -- 19.78kb -> 18.96kb (4.14%)
/assets/images/img_cat/horse.png -- 17.15kb -> 16.45kb (4.08%)
/assets/js/lightbox/images/next.png -- 1.32kb -> 1.27kb (3.85%)
/assets/images/img_cat/tractor.png -- 19.80kb -> 19.14kb (3.33%)
/assets/images/img_cat/cd.png -- 17.42kb -> 16.85kb (3.26%)
/assets/images/img_cat/guitar.png -- 18.86kb -> 18.30kb (2.95%)
/assets/images/img_cat/jewelry.png -- 18.03kb -> 17.53kb (2.76%)
/assets/images/img_cat/hifi.png -- 16.69kb -> 16.23kb (2.73%)
/assets/images/nophoto.jpg -- 3.47kb -> 3.39kb (2.5%)
/assets/images/logoModule.png -- 3.84kb -> 3.76kb (2.14%)
/assets/images/img_cat/paragliding.png -- 10.56kb -> 10.33kb (2.1%)
/assets/images/img_cat/car.png -- 18.60kb -> 18.22kb (2.03%)
/assets/images/img_cat/wine.png -- 16.84kb -> 16.52kb (1.9%)
/assets/images/img_cat/home.png -- 12.65kb -> 12.45kb (1.54%)
/assets/images/img_cat/computer.png -- 13.86kb -> 13.70kb (1.18%)
/assets/images/img_cat/caravan.png -- 16.81kb -> 16.61kb (1.16%)
/assets/images/img_cat/clothing.png -- 16.14kb -> 15.98kb (1%)
/assets/images/img_cat/televisor.png -- 12.08kb -> 11.98kb (0.84%)
/assets/images/img_cat/boat.png -- 12.88kb -> 12.79kb (0.69%)
/assets/images/img_cat/telephony.png -- 9.26kb -> 9.21kb (0.56%)
/assets/images/img_cat/wake.png -- 16.64kb -> 16.57kb (0.46%)
/assets/images/bubble/bubble_ads.png -- 3.84kb -> 3.83kb (0.23%)

mambax7 committed:
Merge branch 'imgbot' of https://github.com/mambax7/adslight

mambax7 committed:
namespaces

mambax7 committed:
::class

mambax7 committed:
static

mambax7 committed:
PhpDocs

mambax7 committed:
RGB to #HEX

mambax7 committed:
Deprecated HTML tag

mambax7 committed:
'strlen(...)' misused

mambax7 committed:
instanceof

mambax7 committed:
'array_push(...)' misused

mambax7 committed:
undefined class

mambax7 committed:
undefined method

mambax7 committed:
remove links to class files

mambax7 committed:
cleanup

mambax7 committed:
center values

mambax7 committed:
23 RC1

mambax7 committed:
postcss

mambax7 committed:
updates

mambax7 committed:
cosmetics

mambax7 committed:
foreach

mambax7 committed:
2.3 RC2

mambax7 committed:
ob_get_clean()

mambax7 committed:
'mkdir(...)' race condition

mambax7 committed:
require

mambax7 committed:
cleanup

mambax7 committed:
CSS shorthand

mambax7 committed:
cleanup

mambax7 committed:
updates

geekwright commented:
I've been looking through the code, and there are some things that might warrant attention, but it is failing in execution tests, so let's concentrate there first.

- Install has failure Utility::saveCategoryPermissions() line 748 (module is not active yet?)
- Install does not create uploads directories, probably due to the above error
- Picture uploads fail with no user visible error, apparently due to missing directories? (My first install had the directories left from previous install. In that case uploads seemed to work fine. There may also be a case issue, as I had two adslight directories in uploads when I cleaned up for a fresh install -- might already be fixed, but can't test yet.)

Longer term suggestions:
- category pictures should move to uploads directory and get an upload function in admin
- jquery dependency should be met with the version included from core (less possibility of conflicts)
- lightbox should be clean and unmodified (improves long term maintainability)
- the ClassifiedsTree related code need to move to the XoopsObjectTree::makeSelectElement() so the select boxes will be themeable

mambax7 commented:
Thanks Richard. I'll be working on it this weekend, but I might just rewrite the whole module, because there is just too much of old code that breaks too easily.  I'll keep you posted...

mambax7 committed:
2.3-RC3

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 7.76kb -> 2.88kb (62.95%)

/assets/images/icons/16/search_button_16.png -- 3.24kb -> 0.83kb (74.4%)
/assets/images/icons/32/discussion.png -- 4.52kb -> 2.05kb (54.76%)

mambax7 commented:
The shorts-term issues seem to be fixed. 
The longer term suggestions in next release, or in the rewrite, if I do so.

mambax7 committed:
Merge pull request #1 from mambax7/imgbot

[ImgBot] Optimize images
